OSDOCS-1117: Add Overview to Scalability and Performance Doc Set (#80080)
openshift/openshift-docs: scalability_and_performance/index.adoc

:_mod-docs-content-type: ASSEMBLY
[id="index"]
= {product-title} scalability and performance overview
include::_attributes/common-attributes.adoc[]
:context: index

// ifndef::openshift-dedicated,openshift-rosa[]

> {product-title} offers best practices and utilities to help you optimize the performance and scale of your {product-title} clusters. The following documentation provides information on recommended performance and scalability practices, reference design specifications, optimization and low latency tuning.

Reviewer: ```suggestion
{product-title} offers best practices and utilities to help you optimize the performance and scale of your {product-title} clusters. The following documentation provides information on recommended performance and scalability practices, reference design specifications, optimization, and low latency tuning.
```

Reviewer: To eliminate first sentence repetition:

```suggestion
{product-title} provides best practices and tools to help you optimize the performance and scale of your clusters. The following documentation provides information on recommended performance and scalability practices, reference design specifications, optimization, and low latency tuning.
```

Author: Done, all good, thank you!

> To engage with Red Hat support, see xref:../support/getting-support.adoc#getting-support[Getting support].

Reviewer: Why "engage" over "contact" or similar?

Reviewer 2: That was my suggestion - after a quick sweep of the repo, it seems I'm on the only one that uses that language :) probably an artifact from my time working in support years ago.

"Contact" is fine too, of course!

Author: Done, thank you.

// endif::openshift-dedicated,openshift-rosa[]

[NOTE]
====

> Some performance and scalability Operators have release cycles that are independent from {product-title} release cycles. For more information see link:access.redhat.com/support/policy/updates/openshift_operators[Openshift Operators].

Reviewer: ```suggestion
Some performance and scalability Operators have release cycles that are independent from {product-title} release cycles. For more information, see link:access.redhat.com/support/policy/updates/openshift_operators[Openshift Operators].
```

Author: Done, thank you.

====

[discrete]

> ==== Recommended performance and scalability practices

Reviewer: This renders as H4. a11y-wise, it should be an H2. Same with other headings past the H1, if I'm guessing right.

Author: Done, thank you!

xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-control-plane-practices.adoc#recommended-control-plane-practices[Recommended control plane practices]

xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-infrastructure-practices.adoc#recommended-infrastructure-practices[Recommended infrastructure practices]

xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc#recommended-etcd-practices[Recommended etcd practices]

[discrete]

> ==== Telco Reference design specifications

Reviewer: ```suggestion
==== Telco reference design specifications
```

Author: Done, yes, thank you.

xref:../scalability_and_performance/telco_ref_design_specs/ran/telco-ran-ref-design-spec.adoc#[Telco RAN DU specification]

xref:../scalability_and_performance/telco_ref_design_specs/core/telco-core-rds-overview.adoc#[Telco core reference design specification]

[discrete]

> ==== Planning, Optimization and Measurement

Reviewer: ```suggestion
==== Planning, optimization, and measurement
```
ISG loves serial commas and sentence case.

Author: Roger that. done!

:_mod-docs-content-type: ASSEMBLY
[id="index"]
= {product-title} Scalability and Performance overview
include::_attributes/common-attributes.adoc[]
:context: index

// ifndef::openshift-dedicated,openshift-rosa[]

> {product-title} offers best practices and utilities to help you optimize the performance and scale of your {product-title} clusters. This book in the documentation provides information on recommended performance and scalability practices, reference design specifications, optimization and low latency tuning.

Reviewer 2: "This book" is self referential language, avoid using when possible. 

"The following documentation provides information on...."

Author: done, thank you!

> To get support on these topics, refer to the section of the documentation entitled xref:../support/getting-support.adoc#getting-support[Getting support]

Reviewer 2: I'd restructure this sentence a bit: 

"To engage with Red Hat support, see [Getting support]." (Don't forget the period at the end, too!)

Author: done, thank you.

// endif::openshift-dedicated,openshift-rosa[]

[NOTE]
====

> Operators described in this section have release cycles that are separate from the core {product-title} release cycles. The release schedule for the Node Observability Operator can be found here link:access.redhat.com/support/policy/updates/openshift_operators[Openshift Operators]

Reviewer 2: This is both too broad and too specific - "Operators described in this section" implies ALL Operators, but you single out the Node Observability Operator in the next sentence. Try this: 

"Some Performance and Scalability Operators have release cycles that are independent from {product-title} release cycles. For more information see, link:...[OpenShift Operator Life Cycles]"

This language is consistent, directs users to the appropriate information, and doesn't single out any Operators.

Author: and included the end-of-sentence period.

====

> The Scalability and Performance section of the documentation offers the following topics:

Reviewer 2: I would personally skip this sentence entirely.

Author: Done. Glad to!

[discrete]
==== Recommended performance and scalability practices

xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-control-plane-practices.adoc#recommended-control-plane-practices[Recommended control plane practices]

xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-infrastructure-practices.adoc#recommended-infrastructure-practices[Recommended infrastructure practices]

xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc#recommended-etcd-practices[Recommended etcd practices]

[discrete]

> ==== Reference design specifications

Reviewer 2: Optional: You could be more specific here, since the links are all Telco RAN: 

==== Telco Reference design specifications

Up to you.

Author: Good point. I was trying to sound general but my examples are from Telco, so I'll be honest about that.
